examples/sandbox/index.ts

```diff
@@ -1269,7 +1269,59 @@ $cosmosIBCTransfer.on("click", async (e) => {
     $cosmosResults.val(label + " does not support Cosmos");
   }
 });
+/*
+ * MAYAChain
+ */
+const $mayachainAddr = $("#mayachainAddr");
+const $mayachainTx = $("#mayachainTx");
+const $mayachainNativeResults = $("#mayachainNativeResults");
 
+$mayachainAddr.on("click", async (e) => {
+  e.preventDefault();
+  if (!wallet) {
+    $mayachainNativeResults.val("No wallet?");
+    return;
+  }
+  // eslint-disable-next-line no-console
+  console.log(core);
+  if (true) {
+    const { addressNList } = wallet.mayachainGetAccountPaths({ accountIdx: 0 })[0];
+    const result = await wallet.mayachainGetAddress({
+      addressNList,
+      showDisplay: false,
+    });
+    await wallet.mayachainGetAddress({
+      addressNList,
+      showDisplay: true,
+    });
+    $mayachainNativeResults.val(result);
+  } else {
+    const label = await wallet.getLabel();
+    $mayachainNativeResults.val(label + " does not support THORChain");
+  }
+});
+$mayachainTx.on("click", async (e) => {
+  e.preventDefault();
+  if (!wallet) {
+    $mayachainNativeResults.val("No wallet?");
+    return;
+  }
+  if (true) {
+    // eslint-disable-next-line no-console
+    console.log("mayachainUnsignedTx: ", mayachainUnsignedTx);
+    const res = await wallet.mayachainSignTx({
+      addressNList: core.bip32ToAddressNList(`m/44'/931'/0'/0/0`),
```

Comment:
This is THORChain derivation path. Isn't MAYAChain path `44'/931'/0'/0/0"` (without `m/`)?

Reply:
Can confirm maya uses exactly the same pathing as thorchain